roles/variables/vars/main.yml

```diff
@@ -152,11 +152,13 @@ kafka_controller_properties:
       listener.security.protocol.map: "{% for listener in kafka_controller_listeners|dict2items %}{% if loop.index > 1%},{% endif %}{{ listener['value']['name'] }}:{{ listener['value'] | confluent.platform.kafka_protocol_defaults(kafka_controller_ssl_enabled, kafka_controller_sasl_protocol)}}{% endfor %}"
       listeners: "{{kafka_controller_listeners['controller']['name']}}://:{{kafka_controller_listeners['controller']['port']}}"
       node.id: "{{ node_id|default(groups.kafka_controller.index(inventory_hostname)|int + 9991) if inventory_hostname in groups.kafka_controller else 0}}"
-  kraft_sasl_plain:
-    enabled: "{{kafka_controller_sasl_protocol=='plain'}}"
-    properties:
-      listener.name.controller.plain.sasl.jaas.config: "{{plain_jaas_config}}"
-      sasl.mechanism.controller.protocol: PLAIN
+  controller_listener_properties:
+    enabled: true
+    properties: "{{ kafka_controller_listeners['controller'] | confluent.platform.client_properties(kafka_controller_ssl_enabled, fips_enabled, kafka_controller_ssl_mutual_auth_enabled, kafka_controller_sasl_protocol,
```

Comment:
Have you verified this? i guess client.properties filter doesn't prefix `listener.name.*`, so that might possibly be an issue.

Reply:
it passed on the rbac-mds scenario, I'll just run ALL on jenkins once